### PR TITLE
Added styling for status error and ok messages

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/css/pelagos-module.css [hunk 1]
```diff
@@ -88,3 +88,37 @@
...
+    border-color: #be7;
+}
+div.status,
+.ok {
+    color: #234600;
+}
+
+div.status, table tr.ok {
+    background-color: #f8fff0;
+}
```
Comment (mickel1138) [inline assets/css/pelagos-module.css]: new line

**Comment by mickel1138** [comment]
We also have : src/Pelagos/Bundle/AppBundle/Resources/public/css/messages.css
Does this do the same thing?

File: assets/css/pelagos-module.css (hunk 1 above)
Comment (mickel1138) [reply assets/css/pelagos-module.css]: This is is also missing warning.

Comment (praneethpr) [reply assets/css/pelagos-module.css]: 👍

**Comment by praneethpr** [comment]
> We also have : src/Pelagos/Bundle/AppBundle/Resources/public/css/messages.css
> Does this do the same thing?

Will move this file to assets folder

File: assets/css/messages.css [hunk 2]
```diff
@@ -7,6 +7,7 @@ div.messages {
 }
 
 div.status {
+  background-image: url("/build/images/message-24-ok.png");
```
Comment (mickel1138) [inline assets/css/messages.css]: Still needs a warning icon?

Comment (mickel1138) [reply assets/css/messages.css]: https://griidc-dev.tamucc.edu/misc/message-24-warning.png

Comment (mickel1138) [reply assets/css/messages.css]: https://griidc-dev.tamucc.edu/misc/message-24-warning.png
Images can be taken from Drupal, so we don't have to add them to our source control.

Comment (praneethpr) [reply assets/css/messages.css]: Added warning image

**Pull request merged**